.github/workflows/pr-analytics.yml

```diff
@@ -0,0 +1,29 @@
+name: Analytics (Pull Request)
+on: pull_request
+env:
+  REALM_DISABLE_ANALYTICS: 1
+jobs:
+  analytics:
+    name: Analytics
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: "recursive"
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 14
+          registry-url: https://registry.npmjs.org/
+      - name: Install npm v7
+        run: npm install -g npm@7
+      # Install the root package (--ignore-scripts to avoid downloading or building the native module)
+      - name: Install root package dependencies
+        run: npm ci --ignore-scripts
+      - name: Install subpackages dependencies
+        run: npm run lerna:bootstrap
+      - name: Run postinstall script
+        run: |
+          cd integration-tests/tests
+          tmpfile=$(mktemp)
+          node ../../scripts/submit-analytics.js --test > $tmpfile
+          cat $tmpfile | grep ^payload | cut -f2- -d' ' | jq '.webHook.properties' | jq 'keys' | jq 'join(",")' | grep -c 'Anonymized Application ID,Anonymized Machine Identifier,Binding,Framework,Framework Version,Host OS Type,Host OS Version,JS Analytics Version,JavaScript Engine,Language,Node.js version,Version,distinct_id,token'
```

Comment:
I'd like the analytics code to be imported from the JS (integration) tests and the `fetchPlatformData` called with different contexts and asserted accordingly.

Reply:
I see your point and we can expand the testing.

The approach here is valuable as it is testing the script as much (including the `package.json` discovery parts).

Reply:
> including the package.json discovery parts.

But it doesn't test the values of the payload properties?